docs/reference/mount.md

```diff
@@ -52,6 +52,58 @@ Searching for lakeFS credentials and server endpoint in the following order:
 - `LAKECTL_*` Environment variables
 - `~/.lakectl.yaml` Configuration file or via `--lakectl-config` flag
 
+## Authenticating with lakeFS (using AWS IAM Role)
+
+Starting from **lakeFS ≥ v1.57.0** and **Everest ≥ v0.4.0**, authenticating with IAM roles is supported!  
+When IAM authentication is configured, Everest will use your machine's **default AWS profile credentials** to generate a **session token** used for authenticating against lakeFS.  
+This token is seamlessly refreshed as long as the AWS session remains valid.  
+For more information regarding IAM authentication, please check [external principals aws](https://docs.lakefs.io/security/external-principals-aws.html).
```

Comment:
No need in that last link sentence (I suggested it 1 line below in prerequisites)